tests/run-tests.sh

```diff
@@ -153,5 +157,5 @@ check_config_files
 setup_workdir
 setup_dss_binary
 check_block_devs "$DEVS"
-run_tests
+run_tests $@
```

Comment:
This is a good idea. Can we push this patch as a separate PR. This is useful even without rest of the stuff.